### PR TITLE
Add Governance Configuration and Chopsticks setup

### PR DESCRIPTION
**Pull request by clangenb**
This PR adds the governance config, namely it adds the technical committee including its membership pallet instance, and adds the democracy pallet. I decided to add the membership pallet after all because during the polkadot update there was a new `EnsureSignedBy` origin check introduced in substrate. This struct made my manual implementation to ensure that the submit proposal origin is a council member obsolete. However, it works only with the membership pallet.

Note: This PR also fixes the pallet-preimage configuration. Submitting a preimage was broken before.

### Chopsticks tests
Fork of the current network and do:
- [x] Check that council and technical committee members are empty
- [x] Set collective members via a sudo call:
    - [x] `councilMembership.resetMembers(Alice, Bob, Charlie)`
    - [x] `technicalCommitteeMembership.resetMembers(Alice, Bob, Charlie)`
- [x] Set the prime of the council via the regular council proposal flow:
   - [x] Council tab -> prose motion `councilMembership.setPrime(Alice)`
   - [x] Vote yes with two members and close motion: tests that > 50% council origin works.
- [x] Fast track some proposal that needs root origin:
   - [x] register preimage via preimage tab: `balances.forceSetBalance(Charlie, 1000000000000)`
   - [x] propose it as an external motion so it can be fast tracked: `council.proposeExternal(preimageHash, preimageLength)`
   - [x]  Fast track it with voting period 10, and delay 5 (random small numbers).
   - [x] go to the developer tap of polkadot.js and run `api.rpc('dev_newBlock', { count: 10 })`
   - [x] check that the referendum has been executed and create another few blocks to execute the scheduled task `api.rpc('dev_newBlock', { count: 5})`
- [x] check that a non-council origin can't submit a proposal

The governance stuff is pretty complex, and more things can be tested, but the above tests are the most crucial things to ensure that we don't lock ourselves out of the system.

### CODE REVIEW

File: runtime/bajun/src/lib.rs [hunk 1]
```diff
@@ -940,21 +918,23 @@ extern crate frame_benchmarking;
 #[cfg(feature = "runtime-benchmarks")]
 mod benches {
 	define_benchmarks!(
+		[cumulus_pallet_xcmp_queue, XcmpQueue]
```
Comment (clangenb) [inline runtime/bajun/src/lib.rs:926]: Sorted the substrate pallet entries here and added the membership benchmarks.

File: runtime/bajun/src/lib.rs [hunk 2]
```diff
@@ -429,7 +436,7 @@ impl pallet_balances::Config for Runtime {
 	type ReserveIdentifier = [u8; 8];
 	type FreezeIdentifier = ();
 	type MaxFreezes = ();
-	type RuntimeHoldReason = ();
+	type RuntimeHoldReason = RuntimeHoldReason;
```
Comment (clangenb) [inline runtime/bajun/src/lib.rs:439]: This broke submitting preimages .

File: runtime/bajun/src/lib.rs [hunk 3]
```diff
@@ -777,19 +748,24 @@ impl pallet_scheduler::Config for Runtime {
...
 	type WeightInfo = weights::pallet_preimage::WeightInfo<Runtime>;
 	type Currency = Balances;
 	type ManagerOrigin = EnsureRoot<AccountId>;
-	type Consideration = HoldConsideration<AccountId, Balances, (), ConvertDeposit>;
+	type Consideration = HoldConsideration<
+		AccountId,
+		Balances,
+		PreimageHoldReason,
+		LinearStoragePrice<PreimageBaseDeposit, PreimageByteDeposit, Balance>,
+	>;
```
Comment (clangenb) [inline runtime/bajun/src/lib.rs:768]: Same config as kusama

File: chopsticks/bajun.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          data:
+            free: 1000000000000000
+      - - - 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty # Bob
+        - providers: 1
+          data:
+            free: 1000000000000000
+      - - - 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y # Charlie
+        - providers: 1
+          data:
+            free: 1000000000000000
```
Comment (clangenb) [inline chopsticks/bajun.yml:30]: Once governance stuff is on the live network, we will add some entries here that override the council and technical committee members with Alice, Bob and Charlie.

**Review by darkfriend77 — APPROVED** [review]
good!

**Comment by darkfriend77** [comment]
Let's move this in place then, for a runtime upgrade.

**Pull request merged**